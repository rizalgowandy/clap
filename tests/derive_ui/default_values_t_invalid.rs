use clap::Parser;

#[derive(Parser, Debug)]
#[command(name = "basic")]
struct Opt {
    #[arg(default_values_t = [1, 2, 3])]
    value: u32,
}

fn main() {
    let opt = Opt::parse();
    println!("{opt:?}");
}
