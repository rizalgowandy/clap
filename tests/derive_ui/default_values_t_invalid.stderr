error: #[arg(default_values_t)] can be used only on Vec types

       = note: see https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

 --> tests/derive_ui/default_values_t_invalid.rs:6:11
  |
6 |     #[arg(default_values_t = [1, 2, 3])]
  |           ^^^^^^^^^^^^^^^^
